packages/augur-ui/src/modules/portfolio/components/common/market-row.styles.less

```diff
@@ -75,6 +75,14 @@
       color: var(--color-secondary-text);
       margin-top: @size-4;
     }
+
+    > span:last-of-type {
```

Comment:
Why are we changing trading UI styles?

Reply:
We are adding a star icon button to the first child div in MarketRowContent. See the red rectangle in the screenshot

![Screen Shot 2020-04-19 at 3 58 29 PM](https://user-images.githubusercontent.com/58647954/79843227-93001c80-83ec-11ea-9911-7ff477832705.png)

 We will hide it in the betting/sport book UI.

Reply:
So those star icons are actually `FavoriteButtons` since the user should be able to toggle if they care about the market anymore. I'm not sure we want to hide them -- I'll bring it up with design since it makes sense to keep it.

Reply:
Sorry. Those are ```FavoriteButtons``` implemented in my PR. 

Sorry for any confusions.

Reply:
They are displayed in the betting/sport book UI.